torch/csrc/jit/serialization/pickler.cpp

```diff
@@ -801,4 +801,87 @@ bool checkHasValidSetGetState(const std::shared_ptr<c10::ClassType>& cls) {
   return true;
 }
 
+// Declare BackendMeta serialization and deserialization function pointer types.
+using BackendMetaPtr =
```

Comment:
`BackendMetaPtr ` is the function pionter form，both for serialization and deserialization